markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -12,56 +12,135 @@ static ros::Publisher flipper_fr_pub;
 static ros::Publisher flipper_rl_pub;
 static ros::Publisher flipper_rr_pub;
 
-static bool flipper_mode_front = false;
-static bool flipper_mode_back = false;
+static bool flipper_mode_fl = false;
+static bool flipper_mode_fr = false;
+static bool flipper_mode_rl = false;
+static bool flipper_mode_rr = false;
 
 static int multiplicator = 0;
 
 void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
 {
   std_msgs::Float64 msg;
-  if (flipper_mode_front == true)
+
+  if (flipper_mode_fl == true)
   {
     msg.data = joy->axes[4] * multiplicator;
     flipper_fl_pub.publish(msg);
+  }
+  if (flipper_mode_fr == true)
+  {
+    msg.data = joy->axes[4] * multiplicator;
     flipper_fr_pub.publish(msg);
   }
-  if (flipper_mode_back == true)
+  if (flipper_mode_rl == true)
   {
     msg.data = joy->axes[4] * multiplicator;
     flipper_rl_pub.publish(msg);
+  }
+  if (flipper_mode_rr == true)
+  {
+    msg.data = joy->axes[4] * multiplicator;
     flipper_rr_pub.publish(msg);
   }
 }
 
 bool flipperModeFrontEnable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
 {
-  flipper_mode_front = true;
-  res.message = "successfully enable flipper mode front";
+  flipper_mode_fl = true;
+  flipper_mode_fr = true;
```

Comment:
I'd add `flipper_mode_rl = false` & `flipper_mode_rr = false`. And add the same but for front flippers on the function `flipperModeBackEnable`.